app/models/pdc_serialization/datacite.rb

```diff
@@ -0,0 +1,137 @@
+# frozen_string_literal: true
+module PDCSerialization
+  # https://support.datacite.org/docs/datacite-metadata-schema-v44-properties-overview
+  class Datacite
+    attr_reader :mapping
+
+    def initialize(mapping)
+      @mapping = mapping
+    end
+
+    # Returns the XML serialization for the Datacite record
+    # Note that the actual XML serialization is done by the datacite-mapping gem.
+    def to_xml
+      @mapping.write_xml
+    end
+
+    ##
+    # Returns the XML serialization for a valid Datacite skeleton record based on a few required values.
+    # Useful for early in the workflow when we don't have much data yet and for testing.
+    #
+    # @param [String] identifier
+    # @param [String] title
+    # @param [String] creator
+    # @param [String] publisher
+    # @param [String] publication_year
+    # @param [String] resource_type
+    def self.skeleton_datacite_xml(identifier:, title:, creator:, publisher:, publication_year:, resource_type:)
+      mapping = ::Datacite::Mapping::Resource.new(
+        identifier: ::Datacite::Mapping::Identifier.new(value: identifier),
+        creators: [] << ::Datacite::Mapping::Creator.new(name: creator),
+        titles: [] << ::Datacite::Mapping::Title.new(value: title),
+        publisher: ::Datacite::Mapping::Publisher.new(value: publisher),
+        publication_year: publication_year,
+        resource_type: datacite_resource_type(resource_type)
+      )
+      mapping.write_xml
+    end
+
+    ##
+    # Creates a PDCSerialization::Datacite object from a Work
+    def self.new_from_work(work)
+      new_from_work_resource(work.resource)
+    end
```

Comment:
I named this `new_from_work` instead of `from_work` (as requested on the original issue) because we use `new_from_x` in `PDCMetadata::Resource`